### DISCUSSION TITLE
ant design vue 里面 FromItem 有一个extra 属性，可以展示一些提示信息， 咋们 n-ui 有没有类似的实现？

### DISCUSSION THREAD

**Discussion by zuihou** [tusen-ai/naive-ui#4322] [Q&A] [answered]
ant design vue 里面 FromItem 有一个extra 属性，可以展示一些提示信息， 咋们 n-ui 有没有类似的实现？

![image](https://user-images.githubusercontent.com/8500410/211266487-8f455865-796d-48b5-9b90-6a4ed2a5aa44.png)

**Comment by XieZongChen** [comment, marked as answer]
暂时没有这种 api

**Reply by zuihou** [reply]
希望能支持一下。  

跟其他ui对比起来，缺了不少功能和api。

祝nui越来越好。

**Reply by XieZongChen** [reply]
可以考虑
